index.html

```diff
@@ -56,6 +57,264 @@ <h2 class="connect">LET'S CONNECT</h2>
+              <li>
+                <a href="#"><img src="images/Vector.svg" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/Linkedin icon.svg" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/Vector-3.svg" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/Vector-2.svg" alt="" /></a>
+              </li>
+              <li>
+                <a href="#"><img src="images/Vector-1.svg" alt="" /></a>
+              </li>
+            </ul>
+          </div>
+          <div class="myself-btn">
+            <a class="resume" href="#">Get my resume</a>
+          </div>
+        </div>
+
+        <div class="skill-list">
+          <div class="frame-12">
+            <div class="frame-7">
+              <nav class="header-bar">
+                <a href="" class="skill-info">Languages</a>
+                <button class="languages-btn">
+                  <svg
+                    width="24"
+                    height="24"
+                    viewBox="0 0 24 24"
+                    fill="none"
+                    xmlns="http://www.w3.org/2000/svg"
+                  >
+                    <path
+                      fill-rule="evenodd"
+                      clip-rule="evenodd"
+                      d="M5.29289 8.29289C5.68342 7.90237 6.31658 7.90237 6.70711 8.29289L12 13.5858L17.2929 8.29289C17.6834 7.90237 18.3166 7.90237 18.7071 8.29289C19.0976 8.68342 19.0976 9.31658 18.7071 9.70711L12.7071 15.7071C12.3166 16.0976 11.6834 16.0976 11.2929 15.7071L5.29289 9.70711C4.90237 9.31658 4.90237 8.68342 5.29289 8.29289Z"
+                      fill="#5E6C84"
+                    />
+                  </svg>
+                </button>
+              </nav>
+            </div>
+            <div class="language-list">
+              <ul class="list-frames">
+                <li class="frame-9">
+                  <img src="images/Ellipse 1.svg" alt="Ellipse 1" /> 
+                  <span>JavScript</span>
+                </li>
+                <li class="frame-9">
+                  <img src="images/Ellipse 1 (1).svg" alt="Ellipse 1 (1)" />
+                  <span>HTML</span>
+                </li>
+                <li class="frame-9">
+                  <img src="images/Ellipse 1 (2).svg" alt="Ellipse 1 (2)" /><span>CSS</span>
+                </li>
+              </ul>
+            </div>
+          </div>
+          <div class="framework-frame">
+            <a href="" class="framework">Frameworks</a>
+            <button class="frame-8-btn">
+              <img src="images/next.png" alt="Vector" />
+            </button>
+          </div>
+          
+          <div class="skills-frame">
+            <a href="" class="skills">Skills</a>
+            <button class="frame-8-btn">
+              <img src="images/next.png" alt="Vector" />
+            </button>
+          </div>
```

Comment:
- [ ] Please make sure that you adjust the style of the dropdown arrows to match the design.

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2022-12-06 16-48-20](https://user-images.githubusercontent.com/92303891/205972493-168c55b1-1d00-45ed-9854-4cd3613c60ce.png)| ![Screenshot from 2022-12-06 16-45-02](https://user-images.githubusercontent.com/92303891/205972284-114c03a6-3298-4e91-b24d-558397801fad.png)|